python/cudf_polars/tests/test_scan.py

```diff
@@ -195,3 +214,28 @@ def test_scan_csv_skip_initial_empty_rows(tmp_path):
     q = pl.scan_csv(tmp_path / "test.csv", separator="|", skip_rows=1)
 
     assert_gpu_result_equal(q)
+
+
+# @pytest.mark.xfail(reason="schema not getting passed through correctly in the polars IR")
+@pytest.mark.parametrize(
+    "schema",
+    [
+        # List of colnames (basicaly like names param in CSV)
+        {"b": pl.String, "a": pl.Float32},
+    ],
+)
+def test_scan_ndjson_schema(df, tmp_path, schema):
+    make_source(df, tmp_path / "file", "ndjson")
+    q = pl.scan_ndjson(tmp_path / "file", schema=schema)
+    # TODO: can remove check_column_order once libcudf read_json supports
+    # usecols
+    assert_gpu_result_equal(q, check_column_order=False)
```

Comment:
*issue*: This one needs to be fixed. Why are we sending things back in the wrong order?